connector/src/test/scala/com/google/cloud/spark/bigquery/SparkBigQueryOptionsSuite.scala

```diff
@@ -35,6 +36,8 @@ class SparkBigQueryOptionsSuite extends FunSuite {
         parameters,
         Map.empty[String, String], // allConf
         hadoopConfiguration,
+        new SQLConf,
+        "2.4.0",
```

Comment:
Please convert to a constant (`SparkVersion`) rather than use a magic String. It applies to the rest of the uses of this String in the other tests